core/templates/dev/head/pages/skill-editor-page/skill-editor-page.controller.ts

```diff
@@ -94,6 +94,35 @@ require('directives/MathjaxBindDirective.ts');
 require('filters/string-utility-filters/normalize-whitespace.filter.ts');
 // ^^^ this block of requires should be removed ^^^
 
+require('objects/templates/BooleanEditorDirective.ts');
+require('objects/templates/CodeStringEditorDirective.ts');
+require('objects/templates/CoordTwoDimEditorDirective.ts');
+require('objects/templates/DragAndDropHtmlStringEditorDirective.ts');
+require('objects/templates/DragAndDropPositiveIntEditorDirective.ts');
+require('objects/templates/FilepathEditorDirective.ts');
+require('objects/templates/FractionEditorDirective.ts');
+require('objects/templates/GraphEditorDirective.ts');
+require('objects/templates/GraphPropertyEditorDirective.ts');
+require('objects/templates/HtmlEditorDirective.ts');
+require('objects/templates/ImageWithRegionsEditorDirective.ts');
+require('objects/templates/IntEditorDirective.ts');
+require('objects/templates/ListOfSetsOfHtmlStringsEditorDirective.ts');
+require('objects/templates/ListOfTabsEditorDirective.ts');
+require('objects/templates/ListOfUnicodeStringEditorDirective.ts');
+require('objects/templates/LogicErrorCategoryEditorDirective.ts');
+require('objects/templates/LogicQuestionEditorDirective.ts');
+require('objects/templates/MathLatexStringEditorDirective.ts');
+require('objects/templates/MusicPhraseEditorDirective.ts');
+require('objects/templates/NonnegativeIntEditorDirective.ts');
+require('objects/templates/NormalizedStringEditorDirective.ts');
+require('objects/templates/NumberWithUnitsEditorDirective.ts');
+require('objects/templates/ParameterNameEditorDirective.ts');
+require('objects/templates/RealEditorDirective.ts');
+require('objects/templates/SanitizedUrlEditorDirective.ts');
+require('objects/templates/SetOfHtmlStringEditorDirective.ts');
+require('objects/templates/SetOfUnicodeStringEditorDirective.ts');
+require('objects/templates/UnicodeStringEditorDirective.ts');
```

Comment:
Could you move these to a **different** separate file (as in `extensions/rich_text_components`) and the require the file here and elsewhere?

Reply:
done!